src/requirements.txt

```diff
@@ -1,26 +1,26 @@
 -i https://pypi.python.org/simple
-asgiref==3.6.0 ; python_version >= '3.7'
-boto3==1.26.69
-botocore==1.29.69 ; python_version >= '3.7'
-cachetools==5.3.0
-certifi==2022.12.7 ; python_version >= '3.6'
+asgiref==3.7.2 ; python_version >= '3.7'
+boto3==1.26.144
+botocore==1.29.144 ; python_version >= '3.7'
+cachetools==5.3.1
+certifi==2023.5.7 ; python_version >= '3.6'
 cfenv==0.5.3
 cffi==1.15.1
-charset-normalizer==3.0.1 ; python_version >= '3.6'
-cryptography==39.0.1 ; python_version >= '3.6'
+charset-normalizer==3.1.0 ; python_full_version >= '3.7.0'
+cryptography==41.0.1 ; python_version >= '3.7'
 defusedxml==0.7.1 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
-dj-database-url==1.2.0
+dj-database-url==2.0.0
 dj-email-url==1.0.6
-django==4.1.6
+django==4.2.1
 django-allow-cidr==0.6.0
-django-auditlog==2.2.2
+django-auditlog==2.3.0
 django-cache-url==3.4.4
 django-csp==3.7
 django-fsm==2.8.1
-django-phonenumber-field[phonenumberslite]==7.0.2
+django-phonenumber-field[phonenumberslite]==7.1.0
 django-widget-tweaks==1.4.12
 environs[django]==9.5.0
-faker==17.0.0
+faker==18.10.0
```

Comment:
🧵 Major version bump

Reply:
No breaking changes listed in https://github.com/joke2k/faker/blob/master/CHANGELOG.md between 17.0.0 and 18.10.0

Reply:
According to their release process, a major version changes should've coincided with breaking changes 🤷🏻 

https://github.com/joke2k/faker/blob/master/RELEASE_PROCESS.rst